File: 1015.py
#!python3.8

# adriano r. de sousa

from math import sqrt

pontos = [[float(x) for x in input().split()], [float(x) for x in input().split()]]

resultado = sqrt(pow(pontos[1][1] - pontos[0][1], 2) + pow(pontos[1][0] - pontos[0][0], 2))

# python3.8
# print(f'{resultado:.4f}')

# python3.4
print('{:.4f}' .format(resultado))

File: 1015.c
#include <stdio.h>
#include <math.h>

typedef struct
{
    double x, y;
} point;

double resultado(point, point);

int main(void){
    point p[2];
    
    scanf("%lf %lf %lf %lf", &p[0].x, &p[0].y, &p[1].x, &p[1].y);

    printf("%.4lf\n", resultado(p[0], p[1]));
    return 0;
}

double resultado(point p1, point p2)
{
    return sqrt(pow(p2.x - p1.x, 2) + pow(p2.y - p1.y, 2));
}